src/main/java/org/mafagafogigante/dungeon/game/PartOfDay.java

```diff
@@ -69,6 +69,24 @@ public static PartOfDay getCorrespondingConstant(Date date) {
     throw new AssertionError(); // Execution should never reach this line.
   }
 
+  /**
+   * Returns the PartOfDay constant corresponding to a given name.
+   *
+   * @param name a name corresponding to PartOfDay enum.
+   * @return a PartOfDay constant.
+   */
+  @NotNull
+  public static PartOfDay getPartOfDayByName(String name) {
```

Comment:
You can annotate this parameter as NotNull too.

Reply:
Alright got it!

```
public static PartOfDay getPartOfDayByName(@NotNull String name) {
```